openlayers-amplify/README.md

```diff
@@ -0,0 +1,49 @@
+# OpenLayers AWS Amplify x Amazon Location Service
+
+![](./img/README01.gif)
+
+![](./img/README02.png)
+
+This example demonstrates how to use OpenLayers with Amazon Location Service and AWS Amplify.
+
+## Execution environment
+
+- node v18.x
+- npm v8.x
+- Amplify CLI v9.2.x
+
+## Getting started
+
+1. Install the Amplify CLI: `npm install -g @aws-amplify/cli`
+2. Install project dependencies: `npm install`
+3. Create a new Amplify project environment: `amplify init`
+4. Create Auth and Geo resources: `amplify push`
+5. After deployment is complete, change the extension of `/src/aws-exports.js` to `/src/aws-exports.ts`
```

Comment:
Let's please remove this step.

Instead, let's add this line to the project's `tsconfig.json`:
```
"allowJs": true,
```

This way the IDE won't complain about the missing types of the `aws-exports` file.

`aws-exports.js` is a file that gets automatically generated and gets overwritten every time the user runs `amplify push`, so better not to rename it.

Reply:
Fixed.